.gitignore

```diff
@@ -6,4 +6,5 @@ MANIFEST
 *.egg-info/
 scapy/VERSION
 test/*.html
+.tox
```

Comment:
Why did you add that?

Reply:
@guedou told me to debug tox using `tox -e flake8` which generates this folder, detected by github

Reply:
OK.